src/bundle/Controller/VersionController.php

```diff
@@ -35,22 +35,19 @@ public function __construct(
 
     public function contentViewTabAction(
         ContentView $view,
-        VersionFilter $versionFilter,
+        VersionParameterSupplier $versionParameterSupplier,
         UiFormFactory $formFactory
     ) {
-        $contentInfo = $view->getContent()->getVersionInfo()->getContentInfo();
-        $versions = $this->uiVersionService->loadVersions($contentInfo);
+        $versionParameterSupplier->supply($view);
 
-        $draftVersions = $versionFilter->filterDrafts($versions);
-        $draftActionsForm = $formFactory->createVersionsDraftActionForm($draftVersions);
-
-        $archivedVersions = $versionFilter->filterArchived($versions);
-        $archivedActionsForm = $formFactory->createVersionsArchivedActionForm($archivedVersions);
+        $draftActionsForm = $formFactory->createVersionsDraftActionForm(
+            $view->hasParameter('draftVersions') ? $view->getParameter('draftVersions') : []
```

Comment:
🤔 not sure there is a better approach without breaking the interface. Did you have something else in mind?
Our views assume that these values are set so isn't it safe for our forms to do the same?